connector/extensions/dcp-impl/src/main/java/org/oaebudt/edc/dcp/policy/MembershipCredentialEvaluationFunction.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
just noticed... what's this copyright? I'm not sure in this project there's the need to set copyright at all, maybe let's sync with the governance and eventually apply in the correct way, for the moment let's get rid of this